packages/app/components/TokenList/index.tsx

```diff
@@ -34,16 +35,18 @@ export const TokenList: FC<Props> = ({
 	};
 
 	return (
-		<FlatList
-			ListHeaderComponent={ListHeaderComponent}
-			showsVerticalScrollIndicator={false}
-			style={style}
-			contentContainerStyle={contentContainerStyle}
-			data={items}
-			renderItem={renderItem}
-			ItemSeparatorComponent={Separator}
-			ListEmptyComponent={ListEmpty}
-		/>
+		<View>
+			<FlatList
+				ListHeaderComponent={ListHeaderComponent}
+				showsVerticalScrollIndicator={false}
+				style={style}
+				contentContainerStyle={contentContainerStyle}
+				data={items}
+				renderItem={renderItem}
+				ItemSeparatorComponent={Separator}
+				ListEmptyComponent={ListEmpty}
+			/>
+		</View>
```

Comment:
check again if my previous comment could solve your problem and revert this file version